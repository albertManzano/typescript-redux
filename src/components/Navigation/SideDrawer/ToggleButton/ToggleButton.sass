.Navigator
    &--line
    width: 90%
    height: 3px
    margin-top: 5px
    background-color: white

@media (max-width: 500px)
    .Navigator--mvl
        display: none

@media (min-width: 499px)
    .Navigator--desktop
        display: none