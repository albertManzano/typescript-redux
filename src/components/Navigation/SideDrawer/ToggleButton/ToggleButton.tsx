import React from 'react'; /* eslint-disable-line */

const ToggleButton = ({ clicked }) => {
	return (
		<div className="Navigator--desktop Navigator__menu" onClick={clicked}>
			<div className="Navigator__menu--line" />
			<div className="Navigator__menu--line" />
			<div className="Navigator__menu--line" />
		</div>
	);
};

export default ToggleButton;
