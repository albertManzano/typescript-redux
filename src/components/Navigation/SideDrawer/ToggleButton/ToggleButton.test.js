import React from 'react';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import ToggleButton from './ToggleButton';

configure({ adapter: new Adapter() });

describe('<ToggleButton /> ', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<ToggleButton/>)
    });

    it('should exist',()=>{
        expect(wrapper.html()).toBeDefined()
    });

    it('should render always',()=>{
        expect(wrapper).toMatchSnapshot()
    })
})
