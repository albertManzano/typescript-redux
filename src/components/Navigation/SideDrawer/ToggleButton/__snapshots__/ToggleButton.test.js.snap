// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToggleButton />  should render always 1`] = `ShallowWrapper {}`;
